nbuild.targets

```diff
@@ -0,0 +1,56 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+	<!-- Reference: https://learn.microsoft.com/en-us/visualstudio/msbuild/walkthrough-using-msbuild?view=vs-2022 -->
+
+	<!-- This file is a sample nbuild.targets to be used in other projects -->
+	<Import Project="$(ProgramFiles)\nbuild\common.targets" />
+
+	<PropertyGroup>
+        <!-- The name of the solution  must be filled.  it's normally the folder where the *.sln resides-->
+		<SolutionName>$([System.IO.Path]::GetFileNameWithoutExtension('$(MSBuildProjectDirectory)'))</SolutionName>
+        <!-- The deployment folder must be filled.  This is the location when the app deployed or installed -->
+        <DeploymentFolder>$(ProgramFiles)\getting-started</DeploymentFolder>
+	</PropertyGroup>
+	
+	<!--Setup the ARTIFACTS folders for binaries and test results-->
+    <Target Name="ARTIFACTS" DependsOnTargets="TAG">
+      <PropertyGroup>
+        <ArtifactsFolder>$(ArtifactsDir)\$(SolutionName)\release\$(ProductVersion)</ArtifactsFolder>
+		<ArtifactsTestResultsFolder>$(ArtifactsDir)\$(SolutionName)\TestResults\$(ProductVersion)</ArtifactsTestResultsFolder>
+      </PropertyGroup>  
+      <ItemGroup>
+            <BinaryFiles Include="
+                        $(SolutionDir)\Release\*.exe;
+                        $(SolutionDir)\Release\*.exe.config;
+                        $(SolutionDir)\Release\*.json;  
+                        $(SolutionDir)\Release\*.dll"
+						 
+                        Exclude="
+						$(SolutionDir)\Release\**\*.pdb;
+						$(SolutionDir)\Release\testhost.*"
+						/>
+
+            <DataFiles Include="
+                       $(SolutionDir)\Release\Data\*.json"
+                      />
+
+            <TestFiles Include="
+                       $(SolutionDir)\Debug\**\*.*;
+                       " />
+
+		    <ArtifactsBlobs Include = "$(ArtifactsFolder)\**\*.*" />
+
+            <RunTimesNetStandard20 Include = "
+								   $(SolutionDir)\Release\netstandard2.0\*.*"
+                                    Exclude="
+						            $(SolutionDir)\Release\**\*.pdb"
+						            />
+		  
+        </ItemGroup>
+		
+		<RemoveDir Directories="$(ArtifactsFolder)" />
+		
+        <Message Text="==> DONE"/>
+    </Target>
+
```

Comment:
@naz-hage should we include another sample target here or is this not necessary?

Reply:
It may be worth adding another target like build and test without changing the tag.  I would call it BUILD_TEST and depends on CLEAN, BUILD, TEST targets.  You can do this in the next pull request.

Reply:
Created #11 for this